src/docs/layouts.md

```diff
@@ -266,4 +266,4 @@ You can disable this behavior with the `setLayoutResolution` Configuration API m
 module.exports = function(eleventyConfig) {
   eleventyConfig.setLayoutResolution(false);
 };
-```
\ No newline at end of file
+```
```

Comment:
Hmm, this is weird. I do not touch this line...

Reply:
Sometimes .editorconfig will normalize the new line on the last line (no big deal!)